### ISSUE TITLE
[Discussion] Ideas to improve deep learning on a particular music style

### ISSUE THREAD

**Issue by antojsan** [deezer/spleeter#195]
Newbie here. First approach to Git, Python/PiP and commands. 

I'm really interested in the development of this tool and the use of it. My main focus is to separate stems in a particular field of music: jazzfunk. I'm not fully aware of how neural networks learn themselves and evolve. So, I need basic info to clarify how to improve Spleeter performance.

Does every song we input to Spleeter makes it gets better results therefore?

Does remaining in a particular style of songs makes it even better with the output results?

Has it progressive learning steps? This meaning if I input "easier" songs, then other a little bit more complicated and then songs more chaotic or freeform, it will learn better than if the first input are complex.

Thanks in advance

(please if anyone can label this into "training", it would be appreciated)

**Comment by aidv** [comment]
I don't think this is the right place to ask the question. I'd ask it in a more general place somewhere. You're asking a very basic ML question, and I recommend watching a few YouTube videos about ML and how AI is trained.

But for a simple explanation: A training model is generated when the AI is trained. The training model is a snapshot of the neural network.

The training model only knows what it has been trained, nothing more.

Spleeter has 3 training models so far.

Yes, you can train your own model and everything is in the Wiki.

No, every song you input does not make Spleeter better.

**Comment by antojsan** [comment]
As an outsider it helps a lot be given this kind of answers to guess more conviniently what to expect using it. I know there's lots of info about that general topic. It's only that using this for the first time with no IT background (just audiophile) seems confusing at times.
